trulens_eval/trulens_eval/react_components/record_viewer/src/utils/types.ts

```diff
@@ -24,9 +24,16 @@ export interface ModuleJSONRaw {
 }
 
 export interface StackJSONRaw {
-  path: {
-    path: ({ item_or_attribute: string } | { index: number })[];
-  };
+  path: string | /* new serialization is just strings */
+  {path: ( /* old json serialization of paths */ 
+    { item: string } | 
+    { items: string[] } | 
+    { attribute: string } |
```

Comment:
@piotrm0 my bad - should we be handling these cases in utils.ts too?

Reply:
I think it may be ok not to. If an issue didn't already come up, it will not in the future given the change to strings.

Reply:
sounds good, thanks!